linux_os/guide/services/ssh/ssh_server/sshd_set_maxstartups/rule.yml

```
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+title: 'Ensure SSH MaxStartups is configured'
+
+description: |-
+    The MaxStartups parameter specifies the maximum number of concurrent
+    unauthenticated connections to the SSH daemon. Additional connections will be
+    dropped until authentication succeeds or the LoginGraceTime expires for a
+    connection. To confgure MaxStartups, you should add or correct the following
+    line in the
+    <tt>/etc/ssh/sshd_config</tt> file:
+    <pre>MaxStartups {{{ xccdf_value("var_sshd_set_maxstartups") }}}</pre>
+
+rationale: |-
+    To protect a system from denial of service due to a large number of pending
+    authentication connection attempts, use the rate limiting function of MaxStartups
+    to protect availability of sshd logins and prevent overwhelming the daemon.
+
+severity: low
+
+identifiers:
+    cce@rhel7: CCE-83610-6
+    cce@rhel8: CCE-83609-8
+
+references:
+    cis@rhel7: 5.3.21
+    cis@rhel8: 5.2.18
```

```suggestion
    cis@rhel8: 5.2.18
    cis@ubuntu2004: 5.2.21
```

While you're here... :-)